src/lib/y2partitioner/widgets/btrfs_modify_button.rb

```diff
@@ -18,45 +18,31 @@
 # find current contact information at www.suse.com.
 
 require "yast"
-require "y2partitioner/widgets/device_menu_button"
+require "y2partitioner/widgets/device_button"
 require "y2partitioner/actions/edit_btrfs"
-require "y2partitioner/actions/edit_btrfs_devices"
 
 module Y2Partitioner
   module Widgets
-    # Menu button for modifying a Btrfs filesystem
-    class BtrfsModifyButton < DeviceMenuButton
+    # Device button for modifying a Btrfs filesystem
+    class BtrfsModifyButton < DeviceButton
```

Comment:
This name is now inconsistent with the rest. It should be `BtrfsEditButton`. There are no other `WhateverModifyButton` any longer. Isn't it?

Reply:
It is

Reply:
In fact, there is already a [`BtrfsEditButton`](https://github.com/yast/yast-storage-ng/blob/master/src/lib/y2partitioner/widgets/btrfs_edit_button.rb). Nice catch :)